src/app/components/applications/SSV/MyAccount/components/Dashboard/Dashboard.tsx

```diff
@@ -114,8 +114,8 @@ const Dashboard = (props: DashboardProps) => {
           <TableBody>
             {rows.map((row, index: number) => (
                 <CustomizedBodyRow
-                    key={index}
-                    onClick={(e : any)=>{rowsAction && rowsAction(index, e);}}
+                    key={ index }
+                    onClick={( e: any )=>{rowsAction && rowsAction(index, e);}}
```

Comment:
Remove unnecessary empty suffix and postfix

Reply:
done